Broken CLI tool: jena.textindexer

### Version

5.6.0-SNAPSHOT

### What happened?

It seems some recent changes to the Cmd system broke at least jena.textindexer. I didn't yet have time to investigate this further yet - probably it's just a minor thing.

With jena-5.5.0:
```
java -jar /opt/apache-jena-fuseki-5.5.0/fuseki-server.jar text.indexer
16:52:51 WARN  system          :: The “SIS_DATA” environment variable is not set.
Dataset name provided but 'no dataset' flag given
```

With a snapshot build:
```
tar xvf apache-jena-fuseki-5.6.0-SNAPSHOT.tar.gz
java -cp fuseki-server.jar jena.textindexer
```

```
Exception in thread "main" java.lang.ExceptionInInitializerError
	at org.apache.jena.sys.JenaSystem.init(JenaSystem.java:88)
	at org.apache.jena.sys.JenaSystem.init(JenaSystem.java:72)
	at org.apache.jena.cmd.Cmds.<clinit>(Cmds.java:33)
	at jena.textindexer.main(textindexer.java:25)
Caused by: java.lang.NullPointerException: Cannot invoke "java.util.Map.put(Object, Object)" because "org.apache.jena.cmd.Cmds.cmds" is null
	at org.apache.jena.cmd.Cmds.injectCmd(Cmds.java:42)
	at org.apache.jena.query.text.cmd.InitTextCmds.cmds(InitTextCmds.java:25)
	at org.apache.jena.query.text.TextQuery.init(TextQuery.java:63)
	at org.apache.jena.query.text.InitJenaText.start(InitJenaText.java:34)
	at org.apache.jena.base.module.Subsystem.lambda$2(Subsystem.java:117)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.apache.jena.base.module.Subsystem.forEach(Subsystem.java:193)
	at org.apache.jena.base.module.Subsystem.forEach(Subsystem.java:169)
	at org.apache.jena.base.module.Subsystem.initialize(Subsystem.java:115)
	at org.apache.jena.sys.JenaSystem$LazyInitializer.jenaSystemInitialization(JenaSystem.java:105)
	at org.apache.jena.sys.JenaSystem$LazyInitializer.<clinit>(JenaSystem.java:96)
```

### Relevant output and stacktrace

```shell

```

### Are you interested in making a pull request?

None

--- a/jena-cmds/src/main/java/org/apache/jena/cmd/Cmds.java
+++ b/jena-cmds/src/main/java/org/apache/jena/cmd/Cmds.java
@@ -30,10 +30,10 @@ import org.apache.jena.sys.JenaSystem;
  */
 public class Cmds {
 
-    static { JenaSystem.init(); }
-
     private static Map<String, Consumer<String[]>> cmds = new ConcurrentHashMap<>();
     private static Object lock = new Object();
+
+    static { JenaSystem.init(); }
 
     // Initialize via JenaSubsystemLifecycle and not rely on class initialization.
     static void init() {}

